site/content/en/guides/introduction/kustomize.md

```diff
@@ -13,7 +13,7 @@ description:
 - Kustomize uses patches to introduce environment specific changes on an already existing standard config file without disturbing it.
 {{< /alert >}}
 
-Kustomize provides a solution for customizing Kubernetes resource configuration free from templates and DSLs.
+[Kustomize](https://kubectl.docs.kubernetes.io/installation/kustomize/) provides a solution for customizing Kubernetes resource configuration free from templates and DSLs.
```

Comment:
Honestly I don't think I'd expect this link to go to the install page, but rather the page explaining Kustomize, i.e. this one! (also, internal links should be relative). If the source of the binary is confusing, perhaps a sentence explicitly stating that it can be installed manually (link to the page on the appropriate phrase in the statement) or used through `kubectl kustomize` would be better.

Reply:
I think the issue is that I was directed to this page when I was looking into kustomize, and I was brought to this page. Note how the URL for the page is https://kubectl.docs.kubernetes.io/guides/introduction/kustomize/. The point of my comment was that someone ending up here could be led to think that kustomize is included in a k8s implementation, just like kubectl discussed in the prior section was.

So I pointed to the install page simply because this page itself didn't talk about that, or convey that it needed to be installed, etc.

But sure, as for where to point and how best to point to it, I would accede to whatever you may choose. I hope now at least you appreciate better WHY I proposed something. :-)